### PR TITLE
Use kudo from master when testing

### PR DESCRIPTION
**Pull request by jbarrick-mesosphere**
Switch from binary download to go run for running kudoctl so we can run from master.

### CODE REVIEW

**Comment by jbarrick-mesosphere** [comment]
Once I get the CLI testing implementation in we shouldn't need this makefile at all.

**Comment by jbarrick-mesosphere** [comment]
>             case.go:105: resource StatefulSet:kudo-test-adjusted-tomcat/zk-zookeeper: .status.readyReplicas: value mismatch: 3 != 2

Let's bump the timeout.

**Review by alenkacz — CHANGES_REQUESTED** [review]
I am not sure we should use master for testing operators. Right now it's a turbulent times and we're breaking things but I think until we release stable version, people should be developing operators against that stable version.

File: .gitignore [hunk 1]
```diff
@@ -13,6 +13,3 @@
 
 # binaries kind and kubectl from make are in bin
 bin
-# from make create-cluster.  We don't store these.
-go.mod
-go.sum
```
Comment (kensipe) [inline .gitignore:18]: I'm not sure we want this... we definitely do not want a run of a test to create a "dirty" env.

**Review by kensipe — APPROVED** [review]
I agree we are breaking things and we shouldn't stop... however I also agree we should test against master at some frequency and have early feedback that the operators are broken with master.   That testing should probably be switched and be in kudo testing operators.   This repo should probably be test against the latest release.

Ideally... PRs for operators should be run against 2 kudo versions.  
1. currently release with a failure stopping a PR
2. against master with pass/fall providing info (or just a warning).   Not PR stopping.

**Comment by alenkacz** [comment]
Agreed Ken that we need these tests, I just don't feel the operators repo should by default point to master. We should have pipeline testing master against operators, but us breaking master in KUDO should not break operators repo (yesterday we discussed that, we don't guarantee master being stable - master can break any time, breakage of master should not mean that people cannot integrate into this repo)

**Comment by jbarrick-mesosphere** [comment]
Closing this one out - 0.4.0 release happens on Monday which will allow us to fix CI.

**Pull request closed without merging**